src/libstd/panicking.rs

```diff
@@ -159,6 +159,22 @@ pub fn take_hook() -> Box<Fn(&PanicInfo) + 'static + Sync + Send> {
 }
 
 /// A struct providing information about a panic.
+///
+/// You can get this object from the [`set_hook()`] function.
```

Comment:
> Firstly this statement sounds like something that might get out of date quickly once we start adding more functions related to PanicInfo.

Even if more methods related to `PanicInfo` get added in the future, it doesn't mean this comment will be inaccurate. I also don't think it's a huge task to update this comment once that happens.

> Secondly, I feel like phrasing of this could be improved. For example:
> ...

Agreed, I like your examples. 👍